po/sv.po

```diff
@@ -1002,17 +967,14 @@ msgstr "503 - Service Otillgänglig: Servern är för närvarande inte tillgäng
 
 #: src/labels.h:557
 msgid "504 - Gateway Timeout: The upstream server failed to send request"
-msgstr ""
-"504 - Gateway Timeout: Uppströms-servern misslyckades med att skicka "
-"förfrågan"
+msgstr "504 - Gateway Timeout: Uppströms-servern misslyckades med att skicka förfrågan"
 
 #: src/labels.h:559
 msgid "505 - HTTP Version Not Supported"
 msgstr "505 - HTTP-version stöds inte"
 
 #: src/labels.h:561
-msgid "509 - Bandwidth Limit Exceeded: The server has exceeded the bandwidth"
-msgstr ""
+msgid "509 - Bandbreddsgräns överskriden: Servern har överskridit bandbredden"
```

Comment:
Thanks for the update. It seems like at line 977 it has a msgid but is missing the corresponding msgstr. This breaks msgmerge, which is used to update translations during the build.